src/xpk/core/kueue.py

```diff
@@ -185,6 +185,23 @@ def install_kueue_on_cluster(args) -> int:
     xpk_print(f'{task} returned ERROR {return_code}')
   return return_code
 
+def wait_for_kueue_available(args) -> int:
```

Comment:
I think it may be good place to raise an Exception instead of returning int. Generally we want to avoid returning int and raise Exceptions where possible

Reply:
Please also note that new code should be type-hinted where possible

Reply:
> I think it may be good place to raise an Exception instead of returning int. Generally we want to avoid returning int and raise Exceptions where possible

Do we have some error handler to use this solution with exception?

Reply:
I can see some error handler and custom exception with error code. But not sure that it is a scope of current task. WDYT?

Reply:
> Please also note that new code should be type-hinted where possible

Done